src/plone/volto/tests/data/6-1.html

```diff
@@ -0,0 +1,14 @@
+<ul>
```

Comment:
Why is there 6-1.html and also 6.html?

Reply:
Because 6.html is a dirty html, while 6-1.html is clean output produced by serializing Slate JSON, so the expectation is that the deserializer ingests the dirty, but produces clean.

Reply:
I see, it does not roundtrip perfectly because the whitespace is normalized.